draft-ietf-quic-multipath.md

```diff
@@ -318,8 +318,11 @@ Path ID 0.
 
 After the handshake concluded if negotiation of multipath support succeeded,
 endpoints SHOULD use PATH_ACK frames instead of ACK frames,
-also for acknowledging so far unacknowledged 0-RTT packets, using
-Path ID 0. Similarly after a successful handshake, endpoints SHOULD also use
+including for acknowledging so far unacknowledged 0-RTT packets using Path ID 0.
+However, even after successful negotiation, endpoints MAY use ACK frames
+to acknowledge 0-RTT packets or 1-RTT packets received on initial path with Path ID 0,
+these ACK frames need to be processed correctly by peer.
```

Comment:
Having read the discussion, I am fine keeping the MAY. An alternative could also be to focus on receiving ACKs (indicate that they must be correctly processed) without explicitly mentioning that the endpoint can send it (I agree this is somewhat mentioned by the SHOULD).

Minor typos.
```suggestion
and these ACK frames need to be processed correctly by the peer.
```